Monika After Story/game/script-topics.rpy

```diff
@@ -14271,3 +14271,32 @@ label monika_hot_springs:
     m 5dkbfb "Ahhh,{w=0.2} {nw}"
     extend 5dkbfa "just the thought of it makes me feel all tingly, [player]~"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_dying_same_day",
+            category=['philosophy', 'romance'],
+            prompt="Dying the same day",
+            random=True,
+            aff_range=(mas_aff.ENAMORED, None),
+            conditional="renpy.seen_label('monika_back_ups')",
+        )
+    )
+
+label monika_dying_same_day:
+    m 1eua "[player], have you ever heard of some couples wishing to die the same day?"
```

Comment:
```suggestion
    m 1eua "[player], have you ever heard of couples wishing to die the same day?"
```
`some` doesn't fit here, it's precluded by `ever`. You'd need something like `I heard of some couples...`